src/components/tooltip/tooltip.jsx

```diff
@@ -21,56 +16,72 @@ export default function Tooltip({
   text,
   truncate,
 }) {
-  const [visible, setVisible] = useState(false);
+  // size of the triangle in pixels; this should be the same as teh value in the linked stylesheet
+  // TODO is there a way to share the variable so its only in one place
+  const triangleHeight = 8;
 
-  if (disabled) return children;
+  const tooltipRef = useRef();
+  const { position, visible, show, hide } = useTooltipPositioning({ tooltipRef, triangleHeight, direction });
 
-  const show = () => setVisible(true);
-  const hide = () => setVisible(false);
+  useLayoutEffect(() => {
+    if (disabled || document.querySelector(`[aria-describedby="${id}"]`)) return;
 
-  const classNames = [className];
+    throw new Error('<Tooltip> was used without an element on the DOM being described by it. ' +
+      'Please add `aria-describedby` to the element this tooltip is being used to describe.');
+  }, []);
```

Comment:
@juanca this more or less what you had in mind?

We could probably also make it `return` early for production as well if we don't want to hinder clients. It should fail tests for devs locally so I expect it won't realistically hit production, but I could be wrong on that.

I could also change it from `throw` to `console.error` or something.

Reply:
This is 💯 

> won't realistically hit production

Yeah, it's okay. We should have caught it by deployment time. The ID lookup is constant so it should be blazingly fast. 

> I could also change it from throw to console.error or something.

I think this is fine. Throwing an error is good because we want the code to stop running and make the developer fix it ASAP. At some point, we will implement proper React error boundary handling so it won't be as intrusive -- i.e. blank page.